src/Symfony/Contracts/HttpClient/Test/Fixtures/web/index.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+if ('cli-server' !== \PHP_SAPI) {
+    // safe guard against unwanted execution
+    throw new \Exception("You cannot run this script directly, it's a fixture for TestHttpServer.");
+}
+
+$vars = [];
+
+if (!$_POST) {
+    $_POST = json_decode(file_get_contents('php://input'), true);
+    $_POST['content-type'] = $_SERVER['HTTP_CONTENT_TYPE'] ?? '?';
+}
+
+foreach ($_SERVER as $k => $v) {
+    switch ($k) {
+        default:
+            if (0 !== strpos($k, 'HTTP_')) {
+                continue 2;
+            }
+            // no break
+        case 'SERVER_NAME':
+        case 'SERVER_PROTOCOL':
+        case 'REQUEST_URI':
+        case 'REQUEST_METHOD':
+        case 'PHP_AUTH_USER':
+        case 'PHP_AUTH_PW':
+            $vars[$k] = $v;
+    }
+}
+
+switch ($vars['REQUEST_URI']) {
+    default:
+        exit;
+
+    case '/':
+    case '/?a=a&b=b':
```

Comment:
is it really useful to have this fixture in contract? Seems arbitrary and something used by the component only.

Same for `TestHttpServer` :confused: